source/config.js

```diff
@@ -249,16 +272,17 @@ Object.defineProperty(Error.prototype, 'toJSON', {
 });
 
 try {
-  module.exports.gitVersion = /.*?(\d+[.]\d+[.]\d+).*/.exec(child_process.execSync('git --version').toString())[1];
+  module.exports.gitVersion = /.*?(\d+[.]\d+[.]\d+).*/.exec(child_process.execSync(module.exports.gitCommand + ' --version').toString())[1];
 } catch (e) {
-  winston.error('Can\'t run "git --version". Is git installed and available in your path?', e.stderr);
+  winston.error('Can\'t run "' + module.exports.gitCommand + ' --version". Is git installed and available in your path?', e.stderr);
+  winston.warn('Alternatively you can set full path to git command via "gitCommand" setting in .ungitrc', e.stderr);
   throw e;
 }
 
 module.exports.ungitPackageVersion = require('../package.json').version;
 
 if (fs.existsSync(path.join(__dirname, '..', '.git'))){
-  const revision = child_process.execSync('git rev-parse --short HEAD', { cwd: path.join(__dirname, '..') })
+  const revision = child_process.execSync(module.exports.gitCommand + ' rev-parse --short HEAD', { cwd: path.join(__dirname, '..') })
```

Comment:
Overlooked it :) Thanks for pointing - somehow it worked when I tested, seems I started ungit with modifed PATH. Confirmed today it doesn't work without fixing that.